api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -162,7 +162,7 @@ private void validateBirthDate(Errors errors, Date birthDate) {
 			return;
 		}
 		rejectIfFutureDate(errors, birthDate, "birthdate");
-		rejectDateIfBefore120YearsAgo(errors, birthDate, "birthdate");
+		rejectDateIfBefore140YearsAgo(errors, birthDate, "birthdate");
```

Comment:
Well done @Ndacyayisenga-droid , can we also write a test to validate this upgrade. Feel free to check through this class  https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/validator/PatientValidatorTest.java   and confirm whether we need to write a test

Reply:
Thank @sherrif10 for the review...... Let me see through it

Reply:
Hey @sherrif10......... Any other thing??? The test passed successfuly